src/index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;700&family=Poppins:wght@300;400;500;600;700&display=swap"
+          rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.scss">
+    <link rel="shortcut icon" href="/src/image/1symbol 2.png" type="image/png">
+
+</head>
+<body>
+
+<header class="header">
+    <div class="header__wrap">
+         <div class="header__left">
+        <div class="header__logo">
+            <img src="/src/image/1symbol 2.png" class="header__logo-image" alt="logo">
+            <span class="header__title">crypter</span>
+        </div>
+        <div class="header__menu">
+            <ul class="header__menu-list">
+                <li class="header__menu-item"><a class="header__menu-link" href="#" >Discover</a></li>
+                <li class="header__menu-item"><a class="header__menu-link" href="#" >How it work</a></li>
+            </ul>
+        </div>
+    </div>
+         <div class="header__right">
+        <div class="header__search">
+            <form>
+                <div class="header__search-input-wrap" ><input type="text" placeholder="Search" class="header__search-input"/> </div>
+            </form>
+        </div>
+        <div class="header__notification-b">
+            <button class="header__notification header__notification--has-notification" title="Open notifications"></button>
+        </div>
+        <div class="header__actions">
+            <a href="#" class="button button--blue header__link">Upload</a>
+            <a href="#" class="button header__link">Connect Wallet</a>
+        </div>
+
+    </div>
+    </div>
+</header>
+
+<div class="hero">
+<div class="hero__wrap">
+<div class="hero__desc">Create, explore, & collect digital art NFTs.</div>
+<h3 class="hero__title">The new creative economy.</h3>
+<a href="#" class="hero__button button">Start your search</a>
+</div>
+</div>
+
+<div class="promo">
+    <div class="promo__wrap">
+        <div class="promo__left">
+            <img src="/src/image/01.png" alt="Art">
+        </div>
+        <div class="promo__right">
+            <h1 class="promo__title">
+                the creator network®
+            </h1>
+            <div class="promo__2badges">
+                <div class="promo__2badges--first">
```

Comment:
You can't use modifier without base class

`promo__2badges promo__2badges--first`